tools/roslaunch/src/roslaunch/rlutil.py

```diff
@@ -89,7 +90,7 @@ def resolve_launch_arguments(args):
     # try to resolve launch file in package first
     if len(args) >= 2:
         try:
-            resolved = roslib.packages.find_resource(args[0], args[1])
+            resolved = cached.find_resource(pkg=args[0], resource_name=args[1])
```

Comment:
I think these are keyword arguments.
https://github.com/ros/ros/blob/jade-devel/core/roslib/src/roslib/packages.py#L463

and cached.py in this package supports both.
https://github.com/ros/ros_comm/pull/676/files#diff-66abf4062401099443c44fc29a06e40aR28
```
cached.find_resource(args[0], args[1])
```
```
cached.find_resource(pkg=args[0], resource_name=args[1])
```